action.yml

```
@@ -0,0 +1,243 @@
+name: 'Release'
+description: 'Create build releases of your create-wordpress-plugin WordPress projects.'
+branding:
+  icon: 'package'
+  color: 'blue'
+inputs:
+  build-command:
+    description: 'Specify the command to run for build.'
+    required: false
+    default: 'npm run build'
+  draft:
+    description: 'Specify whether to create a draft release.'
+    required: false
+    default: 'false'
+  github-token:
+    description: 'Specify the GitHub token to use.'
+    required: false
+    default: ''
```

should this match the readme?
```suggestion
    default: ${{ github.token }}
```

---

I was doing the check down below but sure